src/pages/Stream/Views/Explore/StaticLogRow.tsx

```diff
@@ -27,21 +67,53 @@ const LogRow: FC<LogRowProps> = (props) => {
 		...headers.filter((header) => (isPinned ? !pinnedColumns.includes(header) : pinnedColumns.includes(header))),
 	];
 	const columnsToShow = headers.filter((header) => !columnsToIgnore.includes(header));
+
+	const handleMouseHover = useCallback((row: number, column: number, isCursorInside: boolean) => {
+		if (!isCursorInside) {
+			setHoveredCell(null);
+		} else {
+			setHoveredCell(`${row}-${column}`);
+		}
+	}, []);
+
 	const onClick = useCallback((log: Log) => {
+		const selectedText = window.getSelection()?.toString();
+		if (selectedText !== undefined && selectedText?.length > 0) return;
+
 		setLogsStore((store) => setSelectedLog(store, log));
 	}, []);
+
 	return (
 		<Fragment>
-			{pageData.map((log, logIndex) => {
-				return (
-					<tr key={logIndex} className={logIndex % 2 ? trStyle : trEvenStyle} onClick={() => onClick(log)}>
-						{columnsToShow.map((header, logSchemaIndex) => {
-							return <td key={`${header}-${logSchemaIndex}`}>{parseLogData(log[header], header)}</td>;
-						})}
-						{rowArrows && <ViewLogArrow />}
-					</tr>
-				);
-			})}
+			{pageData.map((log, logIndex) => (
+				<tr key={logIndex} className={logIndex % 2 ? trStyle : trEvenStyle} onClick={() => onClick(log)}>
+					{columnsToShow.map((header, logSchemaIndex) => {
+						const parsedData = parseLogData(log[header], header);
+
+						return (
+							<td
+								key={`${header}-${logSchemaIndex}`}
+								onMouseEnter={() => handleMouseHover(logIndex, logSchemaIndex, true)}
+								onMouseLeave={() => handleMouseHover(0, 0, false)}>
```

Comment:
if you are clearing the state on mouse leave, why are you sending 0,0 as params if it only depends on the third argument?
Update the params of handleMouseHover.

Reply:
done